src/templates/static/legal/tac_sections/affiliate.jsx

```diff
@@ -3,121 +3,121 @@ import ListNested from '../../../_common/components/list_nested.jsx';
-                    { text: it.L('The Company will provide advertising collaterals with links containing affiliate IDs which may be modified by us from time to time.') },
-                    { text: it.L('With regards to tracking your referrals accessing [_1] via the links on your website or by quoting your affiliate IDs during the sign-up process, the Company shall use all reasonable endeavors to ensure that whenever a client is referred to [_1] through these links or affiliate IDs, and they subsequently place a contract or otherwise transact with the Company, the relevant client is identified as originating from your referral. However, the Company shall not be liable to you in any way if the Company is unable to identify a client as originating from your referral. Only properly tagged clients can be assigned to you. It is your responsibility to ensure that all links are properly tagged.', it.website_name) },
+                    { text: it.L('The Affiliate and the Company are independent contractors, and nothing in this Agreement will create any partnership, joint venture, agency, franchise, sales representative, or employment relationship between them. The Affiliate will have no authority to make or accept any offers or representations on the Company\'s behalf. The Affiliate will not make any statement, whether on the Affiliate\'s website or otherwise, that would reasonably contradict anything in this paragraph.') },
+                    { text: it.L('The Company reserves the right to refuse the Affiliate\'s referrals\' application to open an account (or to close the Affiliate\'s account), as deemed necessary, to comply with any requirements that the Company may periodically establish and/or that may be required under applicable laws, rules, and regulations.') },
+                    { text: it.L('The Company will provide advertising collaterals with links containing affiliate IDs which may be modified by the Company from time to time.') },
+                    { text: it.L('With regards to tracking the Affiliate\'s referrals accessing [_1] via the links on the Affiliate\'s website or by quoting affiliate IDs during the sign-up process, the Company shall use all reasonable endeavours to ensure that whenever a client is referred to [_1] through these links or affiliate IDs, and they subsequently place a contract or otherwise transact with the Company, the relevant client is identified as originating from the Affiliate\'s referral. However, the Company shall not be liable to the Affiliate in any way if the Company is unable to identify a client as originating from the Affiliate\'s referral. Only properly tagged clients can be assigned to the Affiliate. It is the Affiliate\'s responsibility to ensure that all links are properly tagged.', it.website_name) },
```

Comment:
```suggestion
                    { text: it.L('With regards to tracking the Affiliate\'s referrals accessing [_1] via the links on the Affiliate\'s website or by quoting affiliate IDs during the sign-up process, the Company shall use all reasonable endeavours to ensure that whenever a client is referred to [_1] through these links or affiliate IDs and they subsequently place a contract or otherwise transact with the Company, the relevant client is identified as originating from the Affiliate\'s referral. However, the Company shall not be liable to the Affiliate in any way if the Company is unable to identify a client as originating from the Affiliate\'s referral. Only properly tagged clients can be assigned to the Affiliate. It is the Affiliate\'s responsibility to ensure that all links are properly tagged.', it.website_name) },
```